static/js/components/Source.jsx

```diff
@@ -63,6 +70,63 @@ const Source = ({ route }) => {
     );
   }
 
+  const groupBy = (array, key) => array.reduce((result, cv) => {
+    (result[cv[key]] = result[cv[key]] || []).push(cv);
+    return result;
+  }, {});
+  function showClassification() {
```

Comment:
We should probably make this a regular component and just pass in the required props

Reply:
done